usb/_interop.py

```diff
@@ -96,5 +96,8 @@ def as_array(data=None):
         # When you pass a unicode string or a character sequence,
         # you get a TypeError if the first parameter does not match
         a = array.array('B')
-        a.fromstring(data) # deprecated since 3.2
+        try:
+            a.fromstring(data)
+        except AttributeError:
+            a.frombytes(data.encode())
```

Comment:
Since `fromstring` was just an alias to `frombytes`, we apparently already required data to be a `bytes`-like object.

Reply:
Oh, great!